src/Orchard.Cms.Web/Startup.cs

```diff
@@ -41,6 +40,9 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
 
             app.UseStaticFiles();
             loggerFactory.AddConsole(Configuration);
```

Comment:
You are right NLog can be used also for console. I will change that on another PR